Deployment/Kombit.Samples.CH.WebsiteDemo/Web.config

```diff
@@ -65,105 +67,138 @@
       </md:ContactPerson>
     </ServiceProvider>
     <RequestedAttributes>
-      <att name="dk:gov:saml:attribute:AssuranceLevel" isRequired="true" />
-      <att name="dk:gov:saml:attribute:SpecVer" isRequired="true" />
-      <att name="dk:gov:saml:attribute:KombitSpecVer" isRequired="true" />
-      <att name="dk:gov:saml:attribute:CvrNumberIdentifier" isRequired="false" />
-      <att name="dk:gov:saml:attribute:Privileges_intermediate" isRequired="false" />
+      <att name="dk:gov:saml:attribute:AssuranceLevel" isRequired="true"/>
```

Comment:
Please ask NGT for the required list. He has it ready now.

Reply:
update with new required list